Yoast/wpseo-news
# CS: Move test double classes to separate directory

## Summary

This PR can be summarized in the following changelog entry:

* _N/A_

## Relevant technical choices:

Move test double classes to separate directory as per the YoastCS guidelines.

## Test instructions

This PR can be tested by following these steps:

* Check Travis and if the unit tests still run without problems, all is fine.
* Alternatively, run the unit tests locally.
